src/store/rootReducer.js

```diff
@@ -0,0 +1,20 @@
+
+import * as actions from './actions';
+
+const initialState = {
+    movies:{}
```

Comment:
why did with default value object and not an array?

Reply:
here movies as data and page object as two keys. name might be misleading i will rename it